scr/startScreen/components/button.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button as MuiButton } from "@mui/material";
```

Comment:
Components folder shouldn't be part of the screen. They are going to be used from the whole app. Move it to src folder.